runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingConfig.java

```diff
@@ -118,4 +118,15 @@ default boolean trawlerNotification()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		position = 8,
+		keyName = "trawlerTimer",
+		name = "Trawler timer in seconds",
```

Comment:
Name and description contradict each other

Reply:
I agree. Any suggestions on what the name should be?

Reply:
Name - "Display trawler timer"
Description - ""Display the time remaining in a game of fishing trawler."